script/build.js

```diff
@@ -111,8 +111,13 @@ smith.destination(`../build/${options.buildtype}`);
 // This lets us access the {{buildtype}} variable within liquid templates.
 smith.metadata({ buildtype: options.buildtype });
 
-// To block an app from production add the following to the below list:
-//  ignoreList.push('<path-to-content-file>');
+// To use:
```

Comment:
I think James merged a separate PR to replace these comments, so you probably didn't pick that up yet.